dotcom-rendering/Containerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:16.19.0-alpine AS dependencies
```

Comment:
Can we add a comment highlighting that this is used for CI/CD only, and not deployed as a container to production?

Reply:
Added!